src/CodingMilitia.PlayBall.GroupManagement.Web/appsettings.development.json

```diff
@@ -6,5 +6,8 @@
   },
   "ConnectionStrings": {
     "GroupManagementDbContext": "server=localhost;port=5432;user id=user;password=pass;database=GroupManagement"
+  },
+  "DbInitializerSettings": {
+    "Environment": "DockerDevelopment"

```

Comment:
Now that I see this, I'm thinking we could have done it in a different way.

Couple of options:
1. (the one I thought originally, but probably wrong 😛) move this to `appsettings.json`, name it `Environments` and get an array of environments.
2. (probably better idea) replace `Environment` with something like `Enabled`. In the `appsettings.json` we set it to `false`, in `appsettings.development.json` we override it with `true`.

What do you think?

Reply:
This is what I understand from above, if incorrect please tell me 😁

`appsettings.json`
```
"DbInitializerSettings": {
  "Environments": [{
    "DockerDevelopment": false
    } , {
    "Staging": false
    } , {
    "Production": false
    }
  ]
} 
```

`appsettings.development.json`
```
"DbInitializerSettings": {
  "Environments": [{
    "DockerDevelopment": true
    } , {
    "Staging": false
    } , {
    "Production": false
    }
  ]
} 
```

I do not know if this is how config works in .net core so please let me know 😜

Reply:
For option 2, my idea was:
`appsettings.json`
```json
"DbInitializerSettings": {
  "Enabled": false
}
```
`appsettings.development.json`
```json
"DbInitializerSettings": {
  "Enabled": true
}
```
Depending on which environment we are, it'll use the correct config file, being enabled or disabled accordingly. After thinking about it, having reference to all environments in the same config, doesn't make that much sense 🙂

Reply:
When we add the Docker environment, we just need to create an `appsettings.dockerdevelopment.json`.

Reply:
sorry but I really can't get the idea 😜. I will just revert my commit then please show me how it's done 😁

Reply:
Ok. If we can find a slot, maybe we can do some pair programming on this one 😉

Reply:
sure! I'd love that. Last time I did that with a coulleage was about 2 years ago